anyrun/src/main.rs

```diff
@@ -391,6 +391,7 @@ fn activate(app: &gtk::Application, runtime_data: Rc<RefCell<Option<RuntimeData>
                     None => {
                         // If nothing is selected select either the top or bottom match based on the input
                         match event.keyval() {
+                            _ if combined_matches.is_empty() => (),
```

Comment:
I would rather have this if gate surrounding the match statement, this syntax could be a bit confusing.

Reply:
Understandable, fixed